process/attendance.md

```diff
@@ -0,0 +1,27 @@
+# Attending a WebAssembly meeting
+
+Meetings are held in-person at the location determined in the meeting
+invitation. The invitation will contain logistics on location, time, address,
+parking, nearby hotels, Wi-Fi, and host contact. This invitation will be sent no
+later than 2 weeks before the meeting, through a pull request on the
+[meetings repository](https://github.com/WebAssembly/meetings) as well as
+through an email to the Community Group's mailing list.
+
+It is expected that a small breakfast, snacks, and beverages will be offered on
```

Comment:
Would requiring pre-registration be tractable in-general?
We happen to have a large space for this one, but knowing somewhat ahead attendance affects food onsite etc.

Did we want to reping the list specifically encouraging registration?